CDP4Reporting/DataSource/CDP4ObjectDataSource.cs

```diff
@@ -0,0 +1,39 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="CDP4ObjectDataSource.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Cozmin Velciu, Adrian Chivu
+//
+//    This file is part of CDP4-IME Community Edition.
+//    The CDP4-IME Community Edition is the RHEA Concurrent Design Desktop Application and Excel Integration
+//    compliant with ECSS-E-TM-10-25 Annex A and Annex C.
+//
+//    The CDP4-IME Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or any later version.
+//
+//    The CDP4-IME Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program. If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4Reporting.DataSource
+{
+    using System.Windows;
+
+    using DevExpress.DataAccess.ObjectBinding;
+
+    public class CDP4ObjectDataSource : ObjectDataSource
+    {
+        public void SetValueByModelCode(string modelCode = null, object Value = null)
+        {
+            MessageBox.Show($"SetModelCode {modelCode} => {Value}");
```

Comment:
This looks like it is not final. Do we need to include this in this version?

Reply:
Yes, I will continue working on this soon. 
Code is being called in a test report I'm using and is called in a very specific way.